### PR TITLE
Added an example for Meteor with raw documentation

### PR DESCRIPTION
**Pull request by chees**
See https://github.com/GoogleCloudPlatform/kubernetes/issues/7103

### CODE REVIEW

**Comment by chees** [comment]
We signed the CLA as a corporation. The name of the company is Q42.

File: examples/meteor/meteor-controller.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            "memory": 500000000,
+            "ports": [{"name": "http-server", "containerPort": 8080, "hostPort": 80}]
+          }]
+        }
+      },
+      "labels": { "name": "meteor" }
+    }
+  },
+  "labels": {"name": "meteor"}
+}
```
Comment (fabioy) [inline examples/meteor/meteor-controller.json]: Please update the json to use the v1beta3 objects. The v1beta1 stuff will be removed soon.

File: examples/meteor/meteor-service.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "apiVersion": "v1beta1",
```
Comment (fabioy) [inline examples/meteor/meteor-service.json]: Ditto, and so on below.

**Comment by jeffmendoza** [comment]
Hi @fabioy, I'm committed to updating the sample to the latest standards, including updating the api version, and making the tutorial platform-agnostic. I wanted to get the sample into the kubernetes repo, so I can work on it without impacting @chees any more.

**Comment by fabioy** [comment]
LGTM (with the promise of update). Filed issue #7179 to track this. Thank you.

**Pull request merged**